autoload/fern/internal/drawer/hover_popup.vim

```diff
@@ -46,23 +51,27 @@ function! s:show() abort
     return
   endif
 
-  let line = getline('.')
-  let width = strdisplaywidth(substitute(line, '[^[:print:]]*$', '', 'g'))
   if has('nvim')
     let s:win = nvim_open_win(nvim_create_buf(v:false, v:true), v:false, {
     \    'relative': 'win',
     \    'bufpos': [line('.') - 2, 0],
-    \    'width': width,
+    \    'width': line_width,
     \    'height': 1,
     \    'noautocmd': v:true,
     \    'style': 'minimal',
     \  })
   else
-    let ui_width = screenpos(0, line('.'), 1).col - win_screenpos(0)[1]
+    let drawer_winid = win_getid()
+    let pos = getcurpos(drawer_winid)
+    let curpos = screenpos(drawer_winid, pos[1], 1)
+
+    " what is my cursor position?
+    " does the line overflow beyond the window?
+    let popup_pos = 0
```

Comment:
This variable should be removed.

Reply:
Ahh yes, I forgot to clean up after myself. Thanks!